guardrails/src/drivers/rest/routers/guardrails.py

```
@@ -0,0 +1,119 @@
+                    "message": {"role": "assistant", "content": response_content},
+                    "finish_reason": "stop",
+                }
+            ],
+            "usage": {
+                "prompt_tokens": 0,
+                "completion_tokens": 0,
+                "total_tokens": 0,
+            },
+        }
+
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)